src/common/schema/application.js

```diff
@@ -47,19 +57,12 @@ module.exports = {
           type: 'string',
           minLength: 1,
           maxLength: 51
-        },
```

Comment:
Yep! The frontend handles the validations _enough_ that I'm happy changing this up, especially since it doesn't give a useful error message on failure.